### PR TITLE
19621 sorting modal in ie

### PR DESCRIPTION
**Pull request by 0xF013**
https://community.openproject.org/work_packages/19621

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://0xf013-fix-19621-sorting-modal-in-ie.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://0xf013-fix-19621-sorting-modal-in-ie.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/assets/stylesheets/content/_modal.sass [hunk 1]
```diff
@@ -91,6 +91,12 @@ $ng-modal-image-width:   $ng-modal-image-height
 
   #modal-sorting
     min-height: 120px
+    .form--row
+      overflow: visible
```
Comment (myabc) [inline app/assets/stylesheets/content/_modal.sass:95]: @0xF013 `overflow: visible` is already set on `.form--row`. Is this overriden somewhere?

File: app/assets/stylesheets/content/_modal.sass [hunk 2]
```diff
@@ -91,6 +91,12 @@ $ng-modal-image-width:   $ng-modal-image-height
...
+      .form--field
+        overflow: visible
```
Comment (myabc) [inline app/assets/stylesheets/content/_modal.sass:97]: @0xF013 ditto.

File: app/assets/stylesheets/content/_modal.sass (hunk 1 above)
Comment (0xF013) [inline app/assets/stylesheets/content/_modal.sass:95]: foundation-apps .grid-block

Comment (myabc) [inline app/assets/stylesheets/content/_modal.sass:95]: bollox

Comment (0xF013) [inline app/assets/stylesheets/content/_modal.sass:95]: @myabc bollox as in 'shit!' or as in 'bullshit!'? Pardon my lack of briton swearing knowledge

Comment (myabc) [inline app/assets/stylesheets/content/_modal.sass:95]: @0xF013 are you using themes in your dev environment?

Comment (0xF013) [inline app/assets/stylesheets/content/_modal.sass:95]: @myabc Nein

**Comment by myabc** [comment]
Closed in favour of #2930

**Pull request closed without merging**